docs/advanced/storageclass.md

```diff
@@ -69,5 +87,33 @@ The `volumeBindingMode` field controls when volume binding and dynamic provision
 1. `Immediate`: Binds and provisions a persistent volume once the PersistentVolumeClaim is created.
 2. `WaitForFirstConsumer`: Binds and provisions a persistent volume once a VM using the PersistentVolumeClaim is created.
 
+![](/img/v1.1/storageclass/customize_tab_vol_binding_mode.png)
+
+
+# Appendix - Use Case
+
+## HDD scenario
```

Comment:
```suggestion
## Appendix - Use Case

### HDD Scenario
```

Reply:
Please do not use more than one level-1 headings in one single document, because level-1 headings will not show in the right sidebar

Reply:
OK, thanks for the tips!